src/System.Management.Automation/namespaces/ProviderBase.cs

```diff
@@ -1384,6 +1385,16 @@ public PSHost Host
             }
         }
 
+        /// <summary>
+        /// Gets the default item separator character for this provider.
+        /// </summary>
+        public virtual char ItemSeparator => Path.DirectorySeparatorChar;
+
+        /// <summary>
+        /// Gets the alternate item separator character for this provider.
+        /// </summary>
+        public virtual char AltItemSeparator => Path.AltDirectorySeparatorChar;
```

Comment:
OK, given the committee's decision, we need to make the following change to this property.
@renehernandez Could you please update the code the test? Thanks!

```
        /// <summary>
        /// Gets the alternate item separator character for this provider.
        /// </summary>
        public virtual char AltItemSeparator =>
#if UNIX
            Utils.Separators.Backslash[0];
#else
            Path.AltDirectorySeparatorChar;
#endif
```

Reply:
@daxian-dbw Will do

Reply:
@daxian-dbw Any updates regarding the latest changes?